lib/cocoapods/target/pod_target.rb

```diff
@@ -356,55 +347,50 @@ def resources_bundle_target_label(bundle_name)
       "#{label}-#{bundle_name}"
     end
 
-    # @param  [Symbol] test_type
-    #         The test type to use for producing the test label.
+    # @param  [Specification] test_spec
+    #         The test spec to use for producing the test label.
     #
     # @return [String] The derived name of the test target.
     #
-    def test_target_label(test_type)
-      "#{label}-#{test_type.capitalize}-Tests"
+    def test_target_label(test_spec)
+      # TODO do not use base_name here
+      "#{label}-#{test_spec.test_type.capitalize}-#{test_spec.base_name}"
```

Comment:
@segiddins you and I had talked about this, what is the best thing to use to make sure the name of a test target is uniq?

Reply:
also is this method used outside of `pod_target_installer.rb`? If no then we can move it there?

Reply:
nevermind on last question seems like it is so we keep it here.